.gitlab-ci.yml

```diff
@@ -275,6 +276,7 @@ build:win64:
     NO_PACKAGE: "1"
     WIN_ARCH: "x86_64"
     TOOLCHAIN_OUTPUT: "util/buildbot/toolchain_mingw64.cmake"
+    NO_MINETEST_GAME: 1
```

Comment:
Since some users use these packages I think MTG should stay here.
(The only thing I meant to suggest in my other comment was `EXISTING_MINETEST_DIR`)